stock_prices/stock_prices.py

```diff
@@ -1,8 +1,14 @@
 #!/usr/bin/python
-
 import argparse
-
 def find_max_profit(prices):
+  max_profit=0
+  for i in range(len(prices)-1):
+    for j in range(i+1,len(prices)):
+      if prices[j]-prices[i]>max_profit:
+        max_profit= prices[j]-prices[i]
+    return max_profit
+
+
   pass
 
 
```

Comment:
Hey David , I'm glad you were able to get stock_prices done. I hope to see more progress today. Reach out if you need help!